lib/quadquizaminos_web/live/contests_live.ex

```diff
@@ -55,10 +56,12 @@ defmodule QuadquizaminosWeb.ContestsLive do
     {:noreply, assign(socket, contests: contests, editing_date?: true)}
   end
 
-  def handle_event("save", %{"key" => "Enter", "value" => contest_name}, socket) do
-    {:noreply, socket |> _create_contest(contest_name)}
+  def handle_event("save", %{"name" => name, "contest_date" => contest_date}, socket) do
```

Comment:
What if no date is selected? Should we default to today? Should we just make it null?

Reply:
We have used null